.travis.yml

```diff
@@ -35,5 +35,7 @@ before_script:
 script:
   # Run PHPCS
   - composer run lint
+  # Output the current phpunit version
+  - vendor/bin/phpunit --version
```

Comment:
I wasn't sure if this was just debugging code or if you meant for this to stay in the build script. Fine either way, its just an additional line of output...

Reply:
Yeah, I decided to leave it in, just in case something like this creeps back in the future.